docs/_data/nakadi-event-bus-api.yaml

```diff
@@ -1442,7 +1442,10 @@ paths:
         - $ref: '#/parameters/SubscriptionId'
         - name: show_time_lag
           in: query
-          description: show consumer time lag
+          description: shows consumer time lag as an optional field.
```

Comment:
I think you need to use the "|" for paragraphs in OpenAPI, please check an example where you can find `description: |`